ceremonies/src/lib.rs

```diff
@@ -408,7 +410,15 @@ impl<T: Config> Module<T> {
 			<BootstrapperRegistry<T>>::insert((cid, cindex), &participant_index, &sender);
 			<BootstrapperIndex<T>>::insert((cid, cindex), &sender, &participant_index);
 			<BootstrapperCount>::insert((cid, cindex), participant_index);
-		} else if is_reputable {
+
+			return Ok(())
+		}
+
+		if cindex == 1 {
```

Comment:
this is not legit. non-bootstrappers should only beballowed after the first successful ceremony has beenbperformed by bootstrappers. the cindex will be different for communities joining later

Reply:
Wow, massive brainfart on my end.